lua/nvim-treesitter/parsers.lua

```diff
@@ -211,36 +228,42 @@ list.dart = {
   install_info = {
     url = "https://github.com/UserNobody14/tree-sitter-dart",
     files = { "src/parser.c", "src/scanner.c" },
-  }
+  },
+  maintainers = {"@akinsho"},
 }
 
 list.rst = {
   install_info = {
     url = "https://github.com/stsewd/tree-sitter-rst",
     files = { "src/parser.c", "src/scanner.c" },
-  }
+  },
+  maintainers = {"@stsewd"},
 }
 
 -- Parsers for injections
 list.regex = {
   install_info = {
     url = "https://github.com/tree-sitter/tree-sitter-regex",
     files = { "src/parser.c" }
-  }
+  },
+  maintainers = {"@theHamsta"},
 }
 
 list.jsdoc = {
   install_info = {
     url = "https://github.com/tree-sitter/tree-sitter-jsdoc",
     files = { "src/parser.c" }
-  }
+  },
+  maintainers = {"@steelsojka"},
 }
 
 list.query = {
   install_info = {
     url = "https://github.com/nvim-treesitter/tree-sitter-query",
     files = { "src/parser.c" }
-  }
+  },
+  readme_name = "Tree-sitter query language",
```

Comment:
What about `name` or `description`? Looks like this could be used outside the readme, like in checkhealth.

Reply:
But what do you do with this info? IMHO no worth maintaining something like that.

Reply:
it just felt too specific

Reply:
It's the only language where `query` is not self-explanatory. No body forbids you to do cool stuff with this in future. But not in this PR (originally to just add links)

Reply:
We can update it later on if needed too, so that's no big deal.